pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -174,7 +174,6 @@ func (p PluginToStatus) Merge() *Status {
 		} else if s.Code() == Unschedulable {
 			hasUnschedulable = true
 		}
-		finalStatus.code = s.Code()
```

Comment:
what if the `s.Code()` is neither `Error`, `UnschedulableAndUnresolvable` or `Unschedulable`?

Reply:
That's impossible, `RunFilterPlugins` checks the status code and will return a new status with  `Error` code if so.

Reply:
Do we need a status code validation here?If it's possible to get an invalid `s.Code()`, this line will make sense only when the invalid code is the last one.

Reply:
This is a generic Merge function, while it is currently only used in RunFilterPlugins, we shouldn't rely that this will indeed be the case in the future. I think we should keep the code as is.

Reply:
the comments has explicitly claimed the error list, 
```
// Merge merges the statuses in the map into one. The resulting status code have the following
// precedence: Error, UnschedulableAndUnresolvable, Unschedulable.
```

As I agree this a generic function, so it might be good to consider to update the comments above and also the line of 177, so the code would be more clearly to read.